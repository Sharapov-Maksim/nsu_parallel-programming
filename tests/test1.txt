64
852814 404131 990939 -883568 592453 99677 594874 452592 454295 601427 608171 492099 -694723 -692567 -821535 -445137 295180 -335687 807122 761286 -460948 -965514 436387 648108 594929 456044 -693368 -536016 -198245 -104469 257482 860066 -752990 -629613 812758 504411 -472471 190397 -374672 -34752 -685983 150021 207777 -693186 995185 -995397 -731777 -373345 -809320 270029 -910502 -276856 -870164 796218 573212 690047 -159378 -141046 -656118 -495133 -470033 -899652 628746 726773 
