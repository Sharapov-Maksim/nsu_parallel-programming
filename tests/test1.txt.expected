64
-995397 -965514 -910502 -899652 -883568 -870164 -821535 -809320 -752990 -731777 -694723 -693368 -693186 -692567 -685983 -656118 -629613 -536016 -495133 -472471 -470033 -460948 -445137 -374672 -373345 -335687 -276856 -198245 -159378 -141046 -104469 -34752 99677 150021 190397 207777 257482 270029 295180 404131 436387 452592 454295 456044 492099 504411 573212 592453 594874 594929 601427 608171 628746 648108 690047 726773 761286 796218 807122 812758 852814 860066 990939 995185 
